src/ThreadPool.zig

```diff
@@ -9,12 +9,12 @@ const ThreadPool = @This();
 lock: std.Mutex = .{},
```

Comment:
Good job on this.  The `ThreadPool` implementation and `WaitGroup` implementation are dead simple now.

Reply:
Still getting a deadlock on Drone CI tho :(